src/entity/User.ts

```diff
@@ -249,7 +250,11 @@ export const addNewUser = async (
 
   return con.transaction(async (entityManager) => {
     const isUniqueUser = await checkUsernameAndEmail(entityManager, data);
-    if (!isUniqueUser) {
+    const isDissalowHandle = await checkDissalowHandle(
```

Comment:
We can merge both calls using `Promise.all`. Not major impact but could be nicer